### DISCUSSION TITLE
[Request] Multivariate model example

### DISCUSSION THREAD

**Discussion by jacmag** [microsoft/finnts#104] [Q&A] [answered]
finn_forecast <- forecast_time_series(
  input_data = m750 [%>%](https://business-science.github.io/modeltime/reference/pipe.html) dplyr::[rename](https://dplyr.tidyverse.org/reference/rename.html)(Date = date), 
  combo_variables = [c](https://rdrr.io/r/base/c.html)("id"), 
  target_variable = "value", 
  date_type = "month", 
  forecast_horizon = 3, 
  run_model_parallel = FALSE,
  models_to_run = [c](https://rdrr.io/r/base/c.html)("arima", "ets", "snaive"))

Hello, can you provide an example like this using a multivariate model xgboost/arima-boost/prophet-boost? Thank you!

**Comment by mitokic** [comment, marked as answer]
You can either set `models_to_run = NULL` to run all models in Finn, including multivariate models like xgboost. Or you can specifically call out models you want ran instead of running all of them by doing something like `models_to_run = c("glmnet", "xgboost", "cubist", "arima-boost")`. 

More information on all models used in Finn can be found on the [documentation site](https://microsoft.github.io/finnts/articles/models-used-in-finnts.html).
